tests/sqlparser_common.rs

```diff
@@ -3586,7 +3586,7 @@ fn parse_set_transaction() {
     // TRANSACTION, so no need to duplicate the tests here. We just do a quick
     // sanity check.
     match verified_stmt("SET TRANSACTION READ ONLY, READ WRITE, ISOLATION LEVEL SERIALIZABLE") {
-        Statement::SetTransaction { modes } => assert_eq!(
+        Statement::SetTransaction { modes, .. } => assert_eq!(
```

Comment:
I think it would be valuable to validate the contents of `session` and `mode` here